### DISCUSSION TITLE
Is there a way to rerun all upstream workflows?

### DISCUSSION THREAD

**Discussion by iamyojimbo** [argoproj/argo-workflows#6366] [Q&A] [answered]
Hi there!

Firstly, thanks for this awesome project!

I was just playing around and wondered: Is there way to see a DAG, and then to click a node and re-run all upstream workflows? I think Airflow supports this feature.

Thanks!

**Comment by terrytangyuan** [comment, marked as answer]
Are you looking for retrying or resubmitting (with `--memoized`)?
1. https://argoproj.github.io/argo-workflows/cli/argo_retry/
2. https://argoproj.github.io/argo-workflows/cli/argo_resubmit/
